src/Direction.java

```diff
@@ -0,0 +1,4 @@
+public enum Direction {

```

Comment:
I feel like an enum is redundant if there's just two values. Maybe we could just use a boolean instead?